src/main/java/net/glowstone/entity/AttributeManager.java

```diff
@@ -132,8 +133,14 @@ public double getPropertyValue(Key key) {
         @Getter
         private final double def;
         @Getter
+        private final double min;
+        @Getter
         private final double max;
 
+        Key(String name, double def, double max) {
```

Comment:
Could you document either the fields or the parameters?

Reply:
Done in d9e3a0be852131a4ebdab4e8177301f871acf629.